src/makeRobotAccountant.js

```diff
@@ -7,6 +7,21 @@
 
 function makeRobotAccountant() {
   // write code here
+  let count = 0;
+
+  const getSum = (firstValue) => {
+    count++;
+
+    return function(secondValue) {
+      if (count > 3 && (count % 2) === 0) {
+        return 'Bzzz... Error!';
+      } else {
+        return firstValue + secondValue;
+      }
```

Comment:
there's no need for `else` keyword here, anyways you return a value if condition is truthful
```suggestion
      if (count > 3 && (count % 2) === 0) {
        return 'Bzzz... Error!';
      }
      
      return firstValue + secondValue;
```